src/develop/blend_gui.c

```diff
@@ -401,8 +401,8 @@ static inline int _blendif_print_digits_default(float value)
 {
   int digits;
   if(value < 0.0001f) digits = 0;
-  else if(value < 0.01f) digits = 2;
-  else if(value < 0.1f) digits = 1;
+  else if(value < 0.01f) digits = 1;
```

Comment:
Are you sure you didn't meant 2 here ?

Reply:
Indeed - well spotted. That's what I get for not pressing save!